app/javascript/lib/study-overview/explore-single.js

```diff
@@ -29,19 +29,22 @@ import {
   addSpatialDropdown, getMainViewOptions, getAnnotParams, handleMenuChange
 } from 'lib/study-overview/view-options'
 
-/** Render violin and scatter plots for the Explore tab's single-gene view */
+/** Render violin and scatter plots for the Explore tab's single-gene view
+ * this also handles multi-gene plots where the genes are collapsed by mean/median
+ */
 async function renderSingleGenePlots(study, gene) {
   clearPlots()
 
   $('#box-plot').html('')
   $('#scatter-plots .panel-body').html('')
+  const consensus = $('#search_consensus').val()
 
   // Draw violin (or box) plot if showing group annotation, or
   // draw scatter plot if showing numeric annotation
   const annotType = getAnnotParams().type
   if (annotType === 'group') {
     $('#distribution-link').html('Distribution')
```

Comment:
Tighter scope for `consensus`:

```suggestion
  // Draw violin (or box) plot if showing group annotation, or
  // draw scatter plot if showing numeric annotation
  const annotType = getAnnotParams().type
  if (annotType === 'group') {
    $('#distribution-link').html('Distribution')
    const consensus = $('#search_consensus').val()
```